frontend/tests/email/mailbox.ts

```diff
@@ -9,13 +9,13 @@ export interface Email {
 export abstract class Mailbox {
   constructor(readonly email: string) { }
 
-  abstract fetchEmails(subject: EmailSubjects): Promise<Email[]>;
+  abstract fetchEmails(subject: EmailSubjects | string, extraText?: string): Promise<Email[]>;
```

Comment:
need to remove extra text here no since you removed it from the other files

Reply:
Fixed in commit 86137084, which also fixes the test failures (which were actually directly related to my forgetting to change the API in this abstract class: `extraText` was no longer used in the calls, but because I didn't change this API the compiler wasn't giving me errors).